presto-kafka/src/main/java/io/prestosql/plugin/kafka/schema/confluent/ConfluentModule.java

```diff
@@ -64,6 +64,7 @@ protected void setup(Binder binder)
         newSetBinder(binder, SchemaProvider.class).addBinding().to(AvroSchemaProvider.class).in(Scopes.SINGLETON);
         newSetBinder(binder, SessionPropertiesProvider.class).addBinding().to(ConfluentSessionProperties.class).in(Scopes.SINGLETON);
         binder.bind(TableDescriptionSupplier.class).toProvider(ConfluentSchemaRegistryTableDescriptionSupplier.Factory.class).in(Scopes.SINGLETON);
+        newMapBinder(binder, String.class, SchemaParser.class).addBinding("AVRO").to(AvroSchemaParser.class).in(Scopes.SINGLETON);
```

Comment:
We don't have to bind them separately, `SchemaRegistryClient#getSchemaProviders` would give a `Map<String, SchemaParser>`

Reply:
This method is defined in `io.confluent.kafka.schemaregistry.client.CachedSchemaRegistryClient#getSchemaProviders`.

Also we need to still to parse the schema into an object that Presto Kafka Connector understands so it could create a Presto table etc.